scripts/retdec.py

```diff
@@ -48,8 +48,8 @@ def do_invoke(self, argv):
             "raw_endian": "big" if is_big_endian() else "little",
         }
 
-        raw_cmd = "{}  -m {} --raw-section-vma {} --raw-entry-point {} -e {} -f plain -a {} -o {} -l {} {} --cleanup"
-        bin_cmd = "{}  -m {} -e {} -f plain -a {} -o {} -l {} {} --cleanup"
+        raw_cmd = "'{}' -m {} --raw-section-vma {} --raw-entry-point {} -e {} -f plain -a {} -o {} -l {} '{}' --cleanup"
+        bin_cmd = "'{}' -m {} -e {} -f plain -a {} -o {} -l {} '{}' --cleanup"
```

Comment:
Same here

Reply:
Good call out. Will work on the patch in a bit